projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -37,13 +37,43 @@ interface PieSemiCircleChartProps extends BaseChartProps< DataPointPercentage[]
 	 * true for clockwise, false for counter-clockwise
 	 */
 	clockwise?: boolean;
+
+	/**
+	 * Size of the chart in pixels
+	 */
+	size?: number;
```

Comment:
I believe we are using only width atm per @retrofox s suggestion.

Reply:
Oh yes, I think you might be right there. I'll check into that - thanks :)